arcgis-ios-sdk-samples/Maps/Apply scheduled updates to preplanned map area/README.md

```diff
@@ -2,17 +2,17 @@
 
 Apply scheduled updates to a downloaded preplanned map area.
 
-![Apply Scheduled Updates to Preplanned Map Area Sample](ApplyScheduledUpdatesToPreplannedMapArea.png)
+![Apply Scheduled Updates to Preplanned Map Area Sample](apply-scheduled-updates-to-preplanned-map-area.png)
 
 ## Use case
 
 With scheduled updates, the author can update the features within the preplanned areas on the service once, and multiple end-users can request these updates to bring their local copies up to date with the most recent state. Importantly, any number of end-users can download the same set of cached updates which means that this workflow is extremely scalable for large operations where you need to minimize load on the server.
 
-This workflow can be used by survey workers operating in remote areas where network connectivity is not available. The workers could download mobile map packages to their individual devices and perform their work normally. Once they regain internet connectivity, the mobile map packages can be updated to show any new features that have been added to the online service.
+This workflow can be used by survey workers operating in remote areas where network connectivity is not available. The workers could download mobile map packages to their individual devices and perform their work normally. Once they regain Internet connectivity, the mobile map packages can be updated to show any new features that have been added to the online service.
 
 ## How to use the sample
 
-Start the app. It will display an offline map, check for available updates, and show update availability and size. Tap 'Apply' to apply the updates to the local offline map and show the results.
+Start the app. It will display an offline map, check for available updates, and show update availability and size. Tap "Apply" to apply the updates to the local offline map and show the results.
```

Comment:
Since all the samples are in one app, the app is already open. Although, it seems like this portion isn't even needed--having to open the sample is a given. That could be the first step of every sample!

```suggestion
Open the sample. It will display an offline map, check for available updates, and show update availability and size. Tap "Apply" to apply the updates to the local offline map and show the results.
```

Reply:
Change to `When the sample loads, ...` to align with other samples. 😅